src/components/charts/bubble-chart/bubble-chart.js

```diff
@@ -114,7 +114,7 @@ BubbleChart.propTypes = {
 
 BubbleChart.defaultProps = {
   theme: {},
-  tooltipClassName: 'bubbleChartTooltip',
```

Comment:
Also keep this classname as it will refer to the class that we use to style the tooltip

Reply:
Now `tooltipClassName` is empty by default